en/vespa-quick-start-cli.html

```diff
@@ -0,0 +1,133 @@
+  <li>Architecture: x86_64</li>
+  <li><b>Minimum 6GB</b> memory dedicated to Docker (the default is 2GB on Macs)</li>
+</ul>
+
+<p>
+This guide is tested with Docker for Mac, Community Edition-18.06.1-ce-mac73 (26764) and
+<em>vespaengine/vespa</em> Docker image built 2018-10-01.
+</p>
+
+
+
+<h2 id="running-vespa-in-docker">Running Vespa in Docker</h2>
+<ol>
+<li>
+<p><strong>Validating your environment:</strong></p>
+<pre>
+$ docker info | grep "Total Memory"
+</pre>
+<p>
+Make sure you see at minimum 6Gb.
+</p>
+</li>
+
+<li>
+<p><strong>Install the Vespa CLI:</strong></p>
+<pre data-test="exec">
+brew install vespa-cli or similar
+</pre>
+</li>
+
+<li>
+<p><strong>Start a Vespa Docker container:</strong></p>
+<pre data-test="exec">
+$ docker run --detach --name vespa --hostname vespa-container \
+  --publish 8080:8080 --publish 19071:19071 \
+  vespaengine/vespa
+</pre>
+<p>
+The port <samp>8080</samp> is published to make the search and feed interfaces
+accessible outside of the docker container,
+19071 is the config server endpoint.
+Only one docker container named <samp>vespa</samp> can run at a time so change the name if needed.
+See the
+<a href="https://github.com/vespa-engine/docker-image/blob/master/Dockerfile">Dockerfile</a> and
+the <a href="https://github.com/vespa-engine/docker-image/blob/master/include/start-container.sh">startup script</a>
+for more insights.
+</p>
+</li>
+
+<li>
+<p><strong>Verify it is ready to use:</strong></p>
+<pre data-test="exec">
+vespa status deploy
+</pre>
+</li>
+
+<li>
+<p><strong>Initialize <code>myapp/</code> to a copy of a
+<a href="https://github.com/vespa-engine/sample-apps">sample</a>
+<a href="cloudconfig/application-packages.html">application package</a>:</strong></p>
+<pre data-test="exec">
+vespa init myapp album-recommendation-selfhosted
+cd myapp
+</pre>
+</li>
+
+<li>
+<p><strong>Deploy it:</strong></p>
+<pre data-test="exec">
+vespa deploy .
+</pre>
+</li>
+
+<li>
+<p><strong>Verify it has started:</strong></p>
+<pre data-test="exec">
+vespa status
+</pre>
+</li>
+
+<li>
+<p><strong>Feed documents:</strong></p>
+<pre data-test="exec">
+$ document post mynamespace/music/docid/1 src/test/resources/A-Head-Full-of-Dreams.json
+$ document post mynamespace/music/docid/2 src/test/resources/Love-Is-Here-To-Stay.json
+$ document post mynamespace/music/docid/3 src/test/resources/Hardwired...To-Self-Destruct.json
+</pre>
+<p>This uses the <a href="reference/document-v1-api-reference.html">/document/v1/ API</a> -
+use <a href="vespa-feed-client.html">vespa-feed-client</a> to feed large amounts of data, fast.</p>
+</li>
+
+<li>
+<p><strong>Issue queries:</strong></p>
+<pre data-test="exec" data-test-assert-contains="Head">
+$ vespa query "select * from music where album contains 'head';"
+$ vespa query "select * from sources * where sddocname contains 'music';" "ranking=rank_albums" "ranking.features.query(user_profile)={{cat:pop}:0.8,{cat:rock}:0.2,{cat:jazz}:0.1}"
```

Comment:
```suggestion
{% raw %}
$ vespa query "select * from music where album contains 'head';"
$ vespa query "select * from sources * where sddocname contains 'music';" "ranking=rank_albums" "ranking.features.query(user_profile)={{cat:pop}:0.8,{cat:rock}:0.2,{cat:jazz}:0.1}"
{% endraw %}
```
Avoid liquid parsing of {{ }}